book/src/appendices/b_migration_notes/rendy_migration.md

```diff
@@ -0,0 +1,239 @@
+# Rendy: Migration Guide
+
+## Audio
+
+* `AudioFormat` no longer exists, you have to use the lower level types -- `Mp3Format`, `WavFormat`, `OggFormat`, `FlacFormat`.
+
+## Assets
+
+* `SimpleFormat` trait has merged into `Format`.
+* `Options` associated type has been removed.
+* `NAME` associated constant is now a method call.
+* `Format<A>` type parameter now takes in `Format<D>`, where `D` is `A::Data`.
+* Implement `import_simple` instead of `import`.
+* `Loader#load` no longer takes in the `Options` parameter.
+
+## Input
+
+* `Bindings<String, String>` is now `Bindings<StringBindings>`.
+* `Bindings<AX, AC>` is now `Bindings<T>`, where `T` is a new type you must implement:
+
+    ```rust
+    pub struct ControlBindings;
+
+    impl BindingTypes for ControlBindings {
+        type Axis = PlayerAxisControl;
+        type Action = PlayerActionControl;
+    }
+    ```
+
+    Diff:
+
+    ```patch
+    -Bindings<PlayerAxisControl, PlayerActionControl>
+    +Bindings<ControlBindings>
+    ```
+
+* `InputBundle` type parameters:
+
+    ```patch
+    -InputBundle::<String, String>::new()
+    +InputBundle::<StringBindings>::new()
+    ```
+
+* `UiBundle` type parameters:
+
+    ```patch
+    +use amethyst::renderer::types::DefaultBackend;
+
+    -UiBundle::<String, String>::new()
+    +UiBundle::<DefaultBackend, StringBindings>::new()
+    ```
+
+## Window
+
+* `DisplayConfig`'s `fullscreen` field is now an `Option<MonitorIdent>`. `MonitorIdent` is `MonitorIdent(u16, String)`. What's that? Good question.
+* `WindowBundle` is now separate from `amethyst_renderer`.
+
+    ```rust
+    use amethyst::window::WindowBundle;
+
+    game_data.with_bundle(WindowBundle::from_config_file(display_config_path))?
+    ```
+
+## Renderer
+
+* `amethyst::renderer::VirtualKeyCode` is now `amethyst::input::VirtualKeyCode`
+* `amethyst::renderer::DisplayConfig` is now `amethyst::window::DisplayConfig`
+* `amethyst::renderer::WindowEvent` is now `amethyst::window::WindowEvent`
+* `amethyst::renderer::Event` is no longer re-exported. Use `winit::Event`
```

Comment:
winit is currently reexported in amethyst root, so renderer::Event is now amethyst::winit::Event.